### PR TITLE
DOC: Add SPDHG papers

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,8 +57,11 @@ ODL has been used in a wide range of research and development with several scien
 ##### Articles
 
 
+
 | Description      |  Code  |
 |------------------|--------|
+| *Faster PET Reconstruction with a Stochastic Primal-Dual Hybrid Gradient Method*. [Article](https://www.spiedigitallibrary.org/conference-proceedings-of-spie/10394/103941O/Faster-PET-reconstruction-with-a-stochastic-primal-dual-hybrid-gradient/10.1117/12.2272946.full?SSO=1) |  |
+| *Stochastic Primal-Dual Hybrid Gradient Algorithm with Arbitrary Sampling and Imaging Applications*. [Arxiv](https://arxiv.org/abs/1706.04957) |  |
```
Comment (kohr-h) [inline README.md]: arXiv

Comment (adler-j) [reply README.md]: This would be a general change!

Comment (kohr-h) [reply README.md]: I'm all for the general change. It's the official writing.

Comment (adler-j) [reply README.md]: Merging this and fixing that later

**Pull request merged**